.circleci/config.yml

```diff
@@ -120,7 +122,9 @@ jobs:
                     - homebrew-v5-
             - run:
                 name: Install macos dependancies - homebrew - make macos-setup-download
```

Comment:
"dependencies" misspelled throughout (sorry, I notice these things)

Reply:
den it is